test/lib/adapters/sendin_blue_adapter_test.exs

```diff
@@ -123,8 +128,37 @@ defmodule Bamboo.SendinBlueAdapterTest do
     assert params["bcc"] == %{"bcc@bar.com" => "BCC"}
   end
 
+  test "deliver/2 correctly formats DATA attachments" do
+    email =
+      new_email(
+        to: "noname@bar.com",
+        attachments: [
+          %Attachment{
+            data: File.read!("./test/support/attachment.png"),
+            content_type: "image/png",
+            filename: Path.basename("attachment1.png")
+          },
+          %Attachment{
+            data: File.read!("./test/support/attachment.png"),
+            content_type: "image/png",
+            filename: Path.basename("attachment2.png")
+          }
+        ]
+      )
+
+    email |> SendinBlueAdapter.deliver(@config)
+    assert_receive {:fake_sendinblue, %{params: params}}
+    assert params["to"] == %{"noname@bar.com" => ""}
+
+    assert params |> get_in(["attachment", "attachment1.png"]) ==
+             File.read!("./test/support/attachment.png") |> Base.encode64()
+
+    assert params |> get_in(["attachment", "attachment2.png"]) ==
+             File.read!("./test/support/attachment.png") |> Base.encode64()
+  end
+
   defp new_email(attrs \\ []) do
     attrs = Keyword.merge([from: "foo@bar.com", to: []], attrs)
-    Email.new_email(attrs) |> Bamboo.Mailer.normalize_addresses
+    attrs |> Email.new_email() |> Bamboo.Mailer.normalize_addresses()
```

Comment:
```suggestion
    Keyword.merge([from: "foo@bar.com", to: []], attrs) |> Email.new_email() |> Bamboo.Mailer.normalize_addresses()
```

Reply:
It was like this in the original library, I'll change it